mantidproject/mantid
# Indirect Reduction - qt Signal warnings

**Description of work.**
This PR removes unwanted run button signals coming from the IndirectDataReduction file. This file has no reason to be communicating with the run button.

**To test:**
1.`Interfaces`->`Indirect`->`Data Reduction`
2. Check the terminal log for any qt signal warnings

The Data Analysis and Tools interfaces should also be checked in the same way to make sure similar signal warnings are no longer present. Tools had the warning:

![image](https://user-images.githubusercontent.com/40830825/53799482-5d6f4a00-3f32-11e9-931d-8d126fe895a4.png)

*No release notes required as these warnings were not in the previous release*

Fixes #24989

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/IndividualTicketTesting/)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
